docs/pages/access-controls/sso/okta.mdx

```diff
@@ -1,107 +1,131 @@
 ---
-title: SSH Authentication With Okta as an SSO provider
-description: How to configure SSH access using Okta for SSO
-h1: SSH Authentication with Okta
+title: Authentication With Okta as an SSO provider
+description: How to configure Teleport access using Okta for SSO
 videoBanner: SM4Am-i8cj4
 ---
 
 ## How to use Okta as a single sign-on (SSO) provider for SSH
 
-This guide will cover how to configure [Okta](https://www.okta.com/) to issue
-SSH credentials to specific groups of users. When used in combination with role
-based access control (RBAC), it allows SSH administrators to define policies
-like:
+This guide covers how to configure [Okta](https://www.okta.com/) to provide
+single sign on (SSO) identities to Teleport Enterprise and Teleport Enterprise
+Cloud. When used in combination with role based access control (RBAC), this allows
+Teleport administrators to define policies like:
 
-- Only members of "DBA" group can SSH into machines running PostgreSQL.
+- Only members of "DBA" group can access into PostgreSQL databases.
 - Developers must never SSH into production servers.
+- Members of the HR group can view audit logs but not production environments.
 
 ## Prerequisites
 
-- Okta account with admin access. Your account must include users and at least two groups.
-- Teleport role with access to maintaining `saml` resources.  This is available in the default `editor` role.
+- An Okta account with admin access. Your account must include users and at
+  least two groups.
 
 (!docs/pages/includes/commercial-prereqs-tabs.mdx!)
 
+- A Teleport role with access to edit and maintain `saml` resources. This is
+  available in the default `editor` role.
+
 (!docs/pages/includes/tctl.mdx!)
 
 (!docs/pages/includes/enterprise/samlauthentication.mdx!)
 
-## Configure Okta
+## Step 1/4. Create & assign groups
+
+### Create Groups
+
+Create two groups: "okta-dev" and "okta-admin". In the Okta dashboard go to the
+navigation bar and click **Directory** -> Groups, then **Add group**:
```

Comment:
@ptgott I originally formatted it this way because "Directory" is bold but groups is not in the Okta nav:

![image](https://user-images.githubusercontent.com/2349184/226625870-54de2e04-a9a8-4091-a3b7-32bdcb836f58.png)

Do you think We should align with external system formatting, or use consistent bold to indicated each nav item in a process?

Reply:
I think using bold consistently helps set the nav flow apart from the main text regardless of how the menu is formatted